action/title.php

```diff
@@ -39,12 +39,26 @@ public function handle_meta(Doku_Event $event, $param) {
         $id = $event->data['page'];
 
         try {
-            $title = new Title($id);
+            $page = new Page($id);
 
             if(!blank($event->data['current']['title'])) {
-                $title->setTitle($event->data['current']['title']);
+                $page->setTitle($event->data['current']['title']);
             } else {
-                $title->setTitle(null);
+                $page->setTitle(null);
+            }
+
+            if(!blank($event->data['current']['last_change']['date'])) {
+                $page->setLastRevision($event->data['current']['last_change']['date']);
+            } else {
+                $page->setLastRevision(null);
+            }
+
+            if(!blank($event->data['current']['last_change']['user'])) {
+                $page->setLastEditor($event->data['current']['last_change']['user']);
+            } elseif (!blank($event->data['current']['last_change']['ip'])) {
+                $page->setLastEditor($event->data['current']['last_change']['ip']);
+            } else {
+                $page->setLastEditor(null);
             }
```

Comment:
The above create three database queries to store the data. I think it makes more sense to have something like this:
```
$page->setTitle(...);
$page->setLastEditor(...);
$page->setLastRevision(...);
$page->save();
```

eg. move the save mechanism out of the set methods.